openlibrary/plugins/openlibrary/js/edit.js

```diff
@@ -458,3 +458,70 @@ export function initEdit() {
         }, 1000);
     }
 }
+
+const isbn_invalid_checksum = 'Invalid Isbn checksum digit';
+const isbn10_wrong_length_or_chars = 'ID must be exactly 10 characters [0-9 or X]. For example 0-19-853453-1 or 0198534531';
+const isbn13_wrong_length_or_chars = 'ID must be exactly 13 characters [0-9]. For example 978-3-16-148410-0 or 9783161484100';
+const invalid_isbn = 'ISBN may be invalid. Click \'Add\' again to submit.'
```

Comment:
```suggestion
```
No need for this since we're adding this string to the template.

Reply:
Just wanted to confirm that the other error strings (invalid checksum, wrong length for isbn10 and isbn13) should *not* be added to HTML as localized strings?

Reply:
I think that this is ok for now.  We don't really have a strategy for localizing string in JS yet.

Is this ready for a review?

Reply:
Yep. Ready for review now. Sorry for the delay.

Reply:
@tfmorris, you're right.  I'm committing to localizing these strings after merging this PR into main.  Will tag you as a stakeholder in the PR.